PodcastGenerator/admin/episodes_edit.php

```diff
@@ -116,7 +115,7 @@
     header('Location: ' . $config['url'] . $config['indexfile'] . $config['link'] . $_GET['name'] . '');
     die();
 
-    error: echo ("");
+    error: echo '<strong><p style="color: red;">' . $error . '</p></strong>';
```

Comment:
Please remove this.
This exists only because PHP has no `pass` keyword like python. It is there to escape from the huge if/else chain. It is better than checking the whole time if a variable is already set.

The error is printed more below.

Also please squash then

Reply:
Sorry, I'm really not a PHP guy.  But I can't see where the error is printed elsewhere, and I know from experience that I wasn't seeing the error printed in my browser (which is why I didn't know why my episode edit was failing).  In fact, I can't find the `$error` variable ever used in this .php file except when its value is being _set_.  Can you help me understand this better?

Reply:
Then error handling doesn’t exist in this file I think. The error section is only there to escape from an if else chain. The name is probably a bit confusing

Reply:
Well, the code is going out of its way to set `$error` in four different errorful conditions that are above the context of the diff shown here.  But I can see how my change wasn't correct -- we shouldn't be printing characters outside the HTML context.  So I've updated the pull request to take a different approach that actually shows the `$error` inside the HTML output when it is present.